Godeps/Godeps.json

```diff
@@ -276,11 +276,6 @@
 			"Rev": "63ce630574a5ec05ecd8e8de5cea16332a5a684d"
 		},
 		{
-			"ImportPath": "github.com/aws/aws-sdk-go/private/protocol/restxml",
```

Comment:
these changes seem unrelated

Reply:
Yes, these changes are all auto generated by the script.
Not sure why the script did this.

Reply:
They are gone now.